scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingService.java

```diff
@@ -349,6 +351,16 @@ public boolean kill() {
 
         listener.schedulerStateUpdated(SchedulerEvent.KILLED);
 
+        // To properly exit the java scheduling process
+        new Thread(() -> {
+            try {
+                Thread.sleep(SCHEDULER_KILL_DELAY);
+            } catch (InterruptedException e) {
+                logger.error("Exception occurred, fail to sleep the scheduler killing thread", e);
```

Comment:
logger.warn("Scheduler.kill() was interrupted while waiting for its configured delay, now terminating the process", e)